src/components/forms/CreateEmailTemplate/CreateEmailTemplate.tsx

```diff
@@ -1,7 +1,140 @@
+      <NHSNotifyBackButton formId='create-email-template-back' action={action}>
+        <input
+          type='hidden'
+          name='emailTemplateName'
+          value={emailTemplateName}
+        />
+        <input
+          type='hidden'
+          name='emailTemplateSubjectLine'
+          value={emailTemplateSubjectLine}
+        />
+        <input
+          type='hidden'
+          name='emailTemplateMessage'
+          value={emailTemplateMessage}
+        />
+      </NHSNotifyBackButton>
+      <div className='nhsuk-grid-column-two-thirds'>
+        <ZodErrorSummary errorHeading={errorHeading} state={state} />
+        <NHSNotifyFormWrapper action={action} formId='create-email-template'>
+          <h1 className='nhsuk-heading-xl' data-testid='page-heading'>
+            {pageHeading}
+          </h1>
+
+          <FormSection>
+            <div className={templateNameError && 'nhsuk-form-group--error'}>
+              <Label htmlFor='emailTemplateName'>{templateNameLabelText}</Label>
+              <HintText>{templateNameHintText}</HintText>
+              <TemplateNameGuidance template={TemplateType.EMAIL} />
+              <TextInput
+                id='emailTemplateName'
+                onChange={emailTemplateNameHandler}
+                value={emailTemplateName}
+                error={templateNameError}
+                errorProps={{ id: 'emailTemplateName-error-message' }}
+                data-testid='emailTemplateName-input'
+              />
+            </div>
+          </FormSection>
+
+          <FormSection>
+            <div
+              className={templateSubjectLineError && 'nhsuk-form-group--error'}
+            >
+              <Label htmlFor='emailTemplateSubjectLine'>
+                {templateSubjectLineLabelText}
+              </Label>
+              <TextInput
+                id='emailTemplateSubjectLine'
+                onChange={emailTemplateSubjectLineHandler}
+                value={emailTemplateSubjectLine}
+                error={templateSubjectLineError}
+                errorProps={{ id: 'emailTemplateSubjectLine-error-message' }}
+                data-testid='emailTemplateSubjectLine-input'
+              />
+            </div>
+
+            <Textarea
+              label={templateMessageLabelText}
+              id='emailTemplateMessage'
+              rows={10}
+              onChange={emailTemplateMessageHandler}
+              value={emailTemplateMessage}
+              error={templateMessageError}
+              errorProps={{ id: 'emailTemplateMessage-error-message' }}
+              data-testid='emailTemplateMessage-input'
+            />
+          </FormSection>
+          <Button type='submit' id='create-email-template-submit-button'>
+            {buttonText}
+          </Button>
+        </NHSNotifyFormWrapper>
+      </div>
+      <div className='nhsuk-grid-column-one-third'>
+        <Personalisation />
+        <MessageFormatting template='EMAIL' />
```

Comment:
Can we reference the type statics here (might also need an update the the MessageFormatting component)

Reply:
added to tidy up https://nhsd-jira.digital.nhs.uk/browse/CCM-6950